docs/polkadot/implementors-guide/guide.md

```diff
@@ -913,17 +913,15 @@ Furthermore, the protocols by which subsystems communicate with each other shoul
 
 #### Description
 
-The Candidate Backing subsystem is engaged in by validators in to contribute to the backing of parachain candidates submitted by other validators.
+The Candidate Backing subsystem determines whether parachain blocks are valid or not. When run as a validator or fisherman, this subsystem actually validates incoming candidates.
```

Comment:
It's true they validate the block, but it's missleading this way.  Instead I'd suggest:

"The Candidate Backing subsystem ensures at least one preliminary validator commits to each parablock's correctness.  These initial backers being slashable if the block later proves invalid gives polkadot a rational threat model during subsequent stages."